lib/modules/login/login_presenter.dart

```diff
@@ -5,6 +5,18 @@ abstract class LoginPresenter
 
 class LoginPresenterImpl extends LoginPresenter
     implements LoginViewDelegate, LoginInteractorDelegate {
+  LoginPresenterImpl() {
+    stateDidInit.voidListen(_didInitState);
+    forgotButtonDidTap.voidListen(_didTapForgotButton);
+    loginButtonDidTap.listen(_didTapLoginButton);
+    [emailTextFieldDidChange, passwordTextFieldDidChange]
+        .combineLatest()
+        .listen(_didChangeEmailOrPasswordText);
+
+    didLogin.voidListen(_didLogin);
+    didFailToLogin.listen(_didFailToLogin);
```

Comment:
I'm a bit curious & actually a question: do you know where we dispose or clear these subscriptions when creating via `listen`? 🤔

Reply:
my bad, I missed adding them to a dispose bag 😁